README.md

```diff
@@ -115,7 +115,7 @@ See also: our own [contributor guide] and the Kubernetes [community page].
 
 - kind supports multi-node (including HA) clusters
 - kind supports building Kubernetes release builds from source
-  - support for make / bash / docker or bazel, in addition to pre-published builds
+  - support for make / bash or docker, in addition to pre-published builds
```

Comment:
this is the same thing currently, we should drop this bullet. can do that nit in a follow up as cleaning up bazel on it's own is worth it